### PR TITLE
etcdserver: tick ElectionTicks before starting when bootstrap new cluster

### PR DESCRIPTION
**Pull request by yichengq**
The behavior accelarates the happen of the first-time leader election,
so the new cluster could elect its leader fast. Technically, it could
help to reduce `electionMs` wait time for the first leader election.

Main usage:
1. Quick start for the local cluster when setting a little longer
election timeout
2. Quick start for the global cluster, which sets election timeout to
its maximum 50s.

for #964

### CODE REVIEW

**Comment by yichengq** [comment]
updated.

This patch doesn't accelerate the case that restart full-down clusters which don't have snapshot yet. This is hard to do because applying entries is done in main loop, and raft cannot accelerate election timeout until itself is added into member list. This case should be just a small part in production.

**Comment by yichengq** [comment]
use uniform helper function 'enableFastBootstrap' to accelarate it.

File: etcdserver/raft.go [hunk 1]
```diff
@@ -195,6 +195,14 @@ func (r *raftNode) resumeSending() {
 	p.Resume()
 }
 
+func enableFastBootstrap(n raft.Node, electionTicks int) {
```
Comment (xiang90) [inline etcdserver/raft.go]: How about advanceTicksForElection? Also we should move the comments above the func.

Comment (yichengq) [inline etcdserver/raft.go]: yep. changed.

**Comment by xiang90** [comment]
LGTM

**Pull request merged**